src/promptflow-devkit/promptflow/_sdk/entities/_run.py

```diff
@@ -716,6 +716,7 @@ def _load_from_source(cls, source: Union[str, Path], params_override: Optional[D
             run_info = json.load(f)
 
         return cls(
+            flow=run_info.get("properties", {}).get("azureml.promptflow.flow_definition_blob_path", None),
```

Comment:
The key should not start with azureml for local run

Reply:
This modification has been deleted.